Index: mods/hextweaks.pw.toml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>name = \"HexTweaks\"\nfilename = \"hextweaks-5.1.0.jar\"\nside = \"both\"\n\n[download]\nurl = \"https://cdn.modrinth.com/data/pim6pG9O/versions/AHWOTFYT/hextweaks-5.1.0.jar\"\nhash-format = \"sha1\"\nhash = \"c1e6d64ab0cb9176e26d05fc1de1eb7647647670\"\n\n[update]\n[update.modrinth]\nmod-id = \"pim6pG9O\"\nversion = \"AHWOTFYT\"\n
===================================================================
diff --git a/mods/hextweaks.pw.toml b/mods/hextweaks.pw.toml
--- a/mods/hextweaks.pw.toml	(revision 57a4777e4a408329fc20c7b6c2a9692ad1917f1c)
+++ b/mods/hextweaks.pw.toml	(date 1740008648576)
@@ -1,13 +1,13 @@
 name = "HexTweaks"
-filename = "hextweaks-5.1.0.jar"
+filename = "hextweaks-5.1.2.jar"
 side = "both"
 
 [download]
-url = "https://cdn.modrinth.com/data/pim6pG9O/versions/AHWOTFYT/hextweaks-5.1.0.jar"
+url = "https://cdn.modrinth.com/data/pim6pG9O/versions/hwowMCSm/hextweaks-5.1.2.jar"
 hash-format = "sha1"
-hash = "c1e6d64ab0cb9176e26d05fc1de1eb7647647670"
+hash = "43f613e22a8689e24e63793a16c6f071601ab0d3"
 
 [update]
 [update.modrinth]
 mod-id = "pim6pG9O"
-version = "AHWOTFYT"
+version = "hwowMCSm"
